### ISSUE TITLE
rm data examples

### ISSUE THREAD

**Issue by huangqingyi-code** [microsoft/rStar#13]
Hi,
In the rm_data_examples.json file,  what do 'neg_comount' and 'pos_comunt' mean?

**Comment by huangqingyi-code** [comment]
what do 'neg_count' and 'pos_count' mean?

**Comment by Swaraj-1925** [comment]
it's like negative reward and positive reward i think

**Comment by Gxy-2001** [comment]
Hi,  

`neg_count` represents the number of negative samples for a given prompt, while `pos_count` represents the number of positive samples. These values are used in [[this line](https://github.com/microsoft/rStar/blob/c7c2c2ea99ecb67262672cec89fb75b3a9254203/train/rm.py#L208)](https://github.com/microsoft/rStar/blob/c7c2c2ea99ecb67262672cec89fb75b3a9254203/train/rm.py#L208) of the training script. The specific computation method is provided in [[this pull request](https://github.com/microsoft/rStar/pull/14)](https://github.com/microsoft/rStar/pull/14).  

As shown in the paper, these values are typically equal to 2 in most cases.  

Hope this helps!

**Comment by huangqingyi-code** [comment]
ok. Thank you for your reply
